index.html

```diff
@@ -37,13 +37,13 @@ <h1>Hey there. I'm Hiwot<span class="h1color"><br>
         </div>
         <div>
         <ul class="socialt">
-            <a href="#"><img src="image/twitter.svg" /></a> &nbsp; &nbsp;
-                  <a href="#"><img src="image/vector.png" /></a> 
-                  <a href="#"><img src="image/Vector (1).png" /></a> &nbsp; &nbsp;
-                  <a href="#"><img src="image/medium.svg" /></a> &nbsp; &nbsp;
-                  <a href="#"><img src="image/github.svg" /></a> &nbsp; &nbsp;
-                  <a href="#"><img src="image/angellist.svg" /></a> &nbsp;
-            
+                  <li><a href="#"><img src="image/twitter.svg" /></a> &nbsp; &nbsp; </li>
+                  <li></li><a href="#"><img src="image/vector.png" /></a> </li>
+                  <li></li><a href="#"><img src="image/Vector (1).png" /></a> &nbsp; &nbsp; </li>
+                  <li></li><a href="#"><img src="image/medium.svg" /></a> &nbsp; &nbsp; </li>
+                  <li></li><a href="#"><img src="image/github.svg" /></a> &nbsp; &nbsp; </li>
+                  <li></li><a href="#"><img src="image/angellist.svg" /></a> &nbsp; </li>
```

Comment:
Good job! 

The first `li` tag is implemented properly, but the other 5 `li` tags are closed twice, once in the beginning and once at the end. So please make sure to remove the first closing `</li>` tag before the `<a>` tag. So that all of them will look like the first one.

Example: 
Correct: 
```HTML
<li>
    <a href="#"></a>
</li>
```
Incorrect:  
```HTML
<li></li>
    <a href="#"></a> 
</li>

```